[FIX] l10n_ar_sale: error en release

pablohmontenegro opened the pull request:
Ticket: 73155
Forzamos la actualización del template report_saleorder_document en módulo "sale" para que pueda aplicarse correctamente este fix https://github.com/ingadhoc/argentina-sale/pull/165

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/argentina-sale/pull/167).

pablohmontenegro committed:
[FIX] l10n_ar_sale: error en release
Ticket: 73155
Forzamos la actualización del template report_saleorder_document en módulo "sale" para que pueda aplicarse correctamente este fix #165

zaoral commented:
@roboadhoc r+
